.eslintrc

```diff
@@ -2,13 +2,24 @@
     "env": {
         "node": true
     },
-    "extends": ["airbnb-base", "prettier"],
+    "extends": [
+        "airbnb-typescript/base",
+        "prettier"
+
+    ],
+    "parser": "@typescript-eslint/parser",
     "parserOptions": {
-        "ecmaVersion": 2019
+        "ecmaVersion": 2019,
+        "project": "./tsconfig.json"
     },
-    "plugins": ["prettier"],
+    "plugins": ["prettier","@typescript-eslint"],
     "root": true,
     "rules": {
+        "@typescript-eslint/no-var-requires": 0,
+        "@typescript-eslint/indent": 0,
+        "@typescript-eslint/naming-convention": 0,
+        "@typescript-eslint/space-before-function-paren": 0,
```

Comment:
Note I disabled these rules in the base eslint (so JS files aren't flagged) Then re-enabled these rules in the *.ts overrides so we can address them as we convert files to ts